src/panels/config/zha/zha-entities.ts

```diff
@@ -0,0 +1,177 @@
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  PropertyValues,
+} from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-button/paper-button";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+import "@polymer/iron-flex-layout/iron-flex-layout-classes";
+import "../../../resources/ha-style";
+
+import { HomeAssistant } from "../../../types";
+import { HassEntity } from "home-assistant-js-websocket";
+import { fireEvent } from "../../../common/dom/fire_event";
+import { ItemSelectedEvent } from "./types";
+import { fetchEntitiesForZhaNode } from "../../../data/zha";
+
+declare global {
+  // for fire event
+  interface HASSDomEvents {
+    "zha-entity-selected": {
+      entity?: HassEntity;
+    };
+  }
+}
+
+export class ZHAEntities extends LitElement {
+  public hass?: HomeAssistant;
+  public showHelp?: boolean;
+  public selectedNode?: HassEntity;
+  private _selectedEntityIndex: number;
+  private _entities: HassEntity[];
+  private _haStyle?: DocumentFragment;
+  private _ironFlex?: DocumentFragment;
+
+  constructor() {
+    super();
+    this._entities = [];
+    this._selectedEntityIndex = -1;
+  }
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      showHelp: {},
+      selectedNode: {},
+      _selectedEntityIndex: {},
+      _entities: {},
+    };
+  }
+
+  protected update(changedProperties: PropertyValues): void {
```

Comment:
I think you mean `updated` ?

Reply:
nope. This is working as intended. Am I doing something I shouldn't be? Hopefully not because I am doing it in a few places and I don't clearly see an alternative...

Reply:
Didn't realize we could also use `update` besides `updated`.

Reply:
Ah, looked into it further. `update` is not supposed to be overridden: https://lit-element.polymer-project.org/guide/lifecycle#update

Use `updated` instead.

Reply:
will do